numpy/core/setup.py

```diff
@@ -963,6 +963,7 @@ def generate_umath_c(ext, build_dir):
 
     config.add_subpackage('tests')
     config.add_data_dir('tests/data')
+    config.add_data_dir('tests/examples')
```

Comment:
This was the missing piece to get tests going. I also checked that the new files are included in the sdist, we do not need to change `MANIFEST.in`